h2o-mapreduce-generic/src/main/java/water/hadoop/h2odriver.java

```diff
@@ -586,6 +599,7 @@ public void registerNode (String ip, int port, int attempt, Socket s) {
           flatfile += val;
           flatfile += "\n";
         }
+        createFlatFile(flatfile);
```

Comment:
I would probably move this to somewhere higher level after the clouding is successfully completed.

If everything succeeds `flatfileContent` is available in a static field

Reply:
Good idea, done